socorro/processor/signature_utilities.py

```diff
@@ -564,19 +538,17 @@ def version(self):
         return '1.0'
 
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
-        return 'AbortMessage' in raw_crash and raw_crash.AbortMessage
+        return bool(raw_crash.get('AbortMessage'))
```

Comment:
The file had other instances of wrapping things in `bool` so I just went with that figuring existing convention is good. But you're right--converting to a bool isn't needed.

I don't have a preference. If someone has a preference, I'm happy to do that.